meta-lmp-base/recipes-bsp/u-boot/u-boot-ostree-scr-fit/boot-common.cmd.in

```diff
@@ -1,6 +1,17 @@
 # common boot handlers
 setenv fio_msg "FIO:"
 
+if test -z "${check_board_closed}"; then setenv check_board_closed "imx_is_closed"; fi;
+if test -z "${check_secondary_boot}"; then setenv check_secondary_boot "imx_secondary_boot"; fi;
+if test -z "${set_primary_boot}"; then setenv set_primary_boot "imx_secondary_boot 0"; fi;
+if test -z "${set_secondary_boot}"; then setenv set_secondary_boot "imx_secondary_boot 1"; fi;
```

Comment:
This may be a little confusing to have imx_ on the firmware commands since this is not an NXP platform. is it possible that these names could be more generic (maybe even using fio_ instead of imx_